hw2/03-count.js

```diff
@@ -1,3 +1,24 @@
 // Add your code here
 
-input.addEventListener('keydown', handleKeyDown);
+function handleKeyDown(){
+	//console.log("test");
+	const txt = document.getElementById('txt');
+	const input = document.getElementById('input');
+	//console.log(txt.innerHTML);
+	//console.log(input.value);
+	//alert("Input Value: " + input.value);
```

Comment:
These comments can be deleted. This was an interesting was to implement this :) it will only highlight full words that match rather than highlighting whatever string is typed in that it can match even if it is part of another word.